### PR TITLE
Summarise Example Mapping blog and provide link

### PR DESCRIPTION
**Pull request by mlvandijk**
As suggested, I've summarized @mattwynne 's blog on [Example Mapping](https://cucumber.io/blog/2015/12/08/example-mapping-introduction) and provided a link. If it's lost too much of it's nuance in summary, please let me know!

### CODE REVIEW

File: content/example-mapping.md [hunk 1]
```diff
@@ -3,4 +3,34 @@ title: Example Mapping
...
+
+- Examples to illustrate these rules are written on a **green** card and placed under the relevant rule.
+
+- Questions that cannot be answered during the session are captured on a **red** card so we can move on with the conversation.
+
+We keep going until the group is satisfied that the scope of the story is clear, or we run out of time.
+
+# More information
+
+For more details, see Matt Wynne's blog on [Example Mapping](https://cucumber.io/blog/2015/12/08/example-mapping-introduction)
```
Comment (olleolleolle) [inline content/example-mapping.md]: End sentence with a period.

**Review by olleolleolle — APPROVED** [review]
This is neat and clear.

**Comment by plaindocs** [comment]
What is with all the external posts going down at once?

This looks great, merge it. :-)

**Comment by olleolleolle** [comment]
CodeShip's blog post is now HTTPS:
https://blog.codeship.com/cucumber-best-practices/

**Comment by olleolleolle** [comment]
This diagram blog post is now _lacking its diagram_, so it should be removed:
http://blog.geekdaily.org/2009/06/cucumber-webrat-who-names-these-things.html

**Pull request merged**